packages/joy-types/src/mint/index.ts

```diff
@@ -0,0 +1,74 @@
+import { getTypeRegistry, u32, u64, u128, Option, Enum} from '@polkadot/types';
+import { Balance, BlockNumber } from '@polkadot/types/interfaces';
+import { JoyStruct } from '../JoyStruct';
+
+export class MintId extends u64 {};
+
+export class Setting extends u128 {};
+export class Adding extends u128 {};
+export class Reducing extends u128 {};
```

Comment:
Would be good to add an extra line here:

```suggestion
export class Reducing extends u128 {};

```